jsapp/scss/components/_kobo.modal.scss

```diff
@@ -655,12 +655,53 @@ $z-modal-x: 10;
     border: 1px solid $divider-color;
   }
 
+  .form-view__cell--add-language-form,
+  .form-view__cell--update-language-form {
+    background: #F8F8F8;
+    padding: 15px;
+    position: relative;
+
+    .form-view__link--close {
+      position: absolute;
+      right: 0px;
+      top: 6px;
+
+      i {
+        font-size: 24px;
+      }
+    }
+  }
+
+  .form-view__cell--add-language-form {
+    max-width: 70%;
+    margin-left: auto;
+  }
+
+  .form-view__form--add-language-fields {
+    display: flex;
+    justify-content: space-between;
+
+    .form-view__cell {
+      &:not(:first-child) {margin-left: 10px;}
+      &.form-view__cell--lang-name {flex: 3;}
+      &.form-view__cell--lang-code {flex: 2;}
+      &.form-view__cell--submit-button {padding-top: 10px;}
+    }
+  }
+
   .ReactTable {
     width: 100%;
 
     .rt-thead.-header {
       box-shadow: none;
       border-bottom: 2px solid $divider-color;
+
+      .form-view__icon-button-edit:hover {
+        &::before, &::after {
+          opacity: 0;
+          visibility: hidden;
```

Comment:
Instead of this CSS hack remove `data-tip` from Header icon :)

Reply:
didn't know if it was needed for other reasons (a11y perhaps)